selenium-system-test/src/test/scala/hts/steps/EmailSteps.scala

```diff
@@ -16,33 +16,25 @@
 
 package hts.steps
 
-class EmailSteps extends Steps {
+import hts.pages.{AuthorityWizardPage, ConfirmDetailsPage, EligiblePage}
+import hts.utils.{Configuration, NINOGenerator}
 
-  Given("""^A user has logged in and is on the confirm details page$"""){ () ⇒
+class EmailSteps extends Steps with NINOGenerator {
 
-  }
-
-  Given("""^A user is on the verification email page$"""){ () ⇒
-
-  }
-
-  When("""^they wish to change their email$"""){ () ⇒
-
-  }
-
-  When("""^they enter in a new email and confirm$"""){ () ⇒
-
-  }
-
-  When("""^they indicate that they wish to provide an alternative email address$"""){ () ⇒
+  var nino: Option[String] = None
```

Comment:
do we want to have a `nino` var here given that we can use the current NINO that's been generated in the `NINOGenerator`? This may not be a fix to make in this PR though since Jackie's working on that stuff now